### DISCUSSION TITLE
Video Insert Format for FB

### DISCUSSION THREAD

**Discussion by efast1568** [s9e/TextFormatter#230] [General]
Hi, Joshy,

Have you had experience with setting the video from FB? 
Currently, when inserting a video based on the width setting, sometimes it occupies too much space and appears too large. 
![螢幕擷取畫面 2024-04-19 123752](https://github.com/s9e/TextFormatter/assets/105265831/678f8e9b-d348-4539-957d-4b37d2063064)

Is it possible to limit the background proportion to something similar to 4:3 when playing videos with non-standard ratios, like the original effect on FB?
![螢幕擷取畫面 2024-04-19 123826](https://github.com/s9e/TextFormatter/assets/105265831/e4324007-ab22-4b38-afac-a863b475e4cd)

**Comment by JoshyPHP** [comment]
The short answer is no, not currently. The Facebook embed is set to be 640 px wide (or 100% of the available space) and only resizes itself vertically. I could create a new iframe that resizes itself in both dimensions as the Imgur iframe does but then you'd see another issue; Some of Facebook embeds do not stick to their actual dimensions or ratio. For instance, [this reel](https://www.facebook.com/reel/873906321076441) tries to resize itself to 640 : 360 even though its actual dimensions are 720 : 1280.
